<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUICompare</class>
 <widget class="QWidget" name="GUICompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GUICompare</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="comapreTab">
      <attribute name="title">
       <string>Compare table</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>135</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>from</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>to</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>buy</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>sell</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>diff</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>%</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>available volume</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pair</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="propertyTab">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <attribute name="title">
       <string>Property</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="1" rowspan="6">
        <widget class="QTableWidget" name="tableCurrencyPair">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>pair</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="selectAllExchange">
         <property name="text">
          <string>select all</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelPercent">
         <property name="text">
          <string>Percent:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QCheckBox" name="notification">
         <property name="text">
          <string>notification</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="3">
        <widget class="QTextEdit" name="message">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;$(from) to $(to) ($(pair)): $(current)%; Buy: $(buy); Sell: $(sell); Available volume: $(volume);&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QCheckBox" name="filter">
         <property name="text">
          <string>hide below level</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="levelVolume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="6">
        <widget class="QTableWidget" name="tableExchange">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>pair</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="selectAllPairs">
         <property name="text">
          <string>select all</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="levelPercent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelVolume">
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QTableWidget" name="helpInformation">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>from</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>to</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>pair</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>current</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>ask</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>bid</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>volume</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>template</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>descripton</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>$(from)</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>exchange from</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>$(to)</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>exchange to</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>$(pair)</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>currency pair</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>$(current)</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>current percent value</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>$(sell)</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>Price exchange from</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>$(buy)</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>Price exchange to</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>$(volume)</string>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>available volume for trade</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="2" colspan="4">
        <widget class="QTableWidget" name="colorIndicator">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>percent</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>color</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>opacity</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="addIndicator">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="removeIndicator">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
